benchmark/benchmark

```diff
@@ -67,14 +66,22 @@ Capybara.register_driver :chrome do |app|
   )
 end
 
+Capybara.register_driver :shimmer do |app|
+  Capybara::Shimmer::Driver.new(app, use_proxy: false, port: 9228)
```

Comment:
If you choose to use the debugger proxy, you can start it up (it listens at 9222 and forwards to 9223). The driver is smart enough to know it should use those ports, so you can just pass `use_proxy: true`.